org.eclipse.m2e.pde.ui/src/org/eclipse/m2e/pde/ui/editor/MavenTargetLocationWizard.java

```diff
@@ -115,10 +117,18 @@ public void createControl(Composite parent) {
 					version.setText(clipboardParser.getVersion());
 					classifier.setText(clipboardParser.getClassifier());
 					type.setText(MavenTargetLocation.DEFAULT_PACKAGE_TYPE);
-					scope.setText(MavenTargetLocation.DEFAULT_DEPENDENCY_SCOPE);
+					scope.setText(clipboardParser.getScope());
 					metadata.setSelection(new StructuredSelection(MavenTargetLocation.DEFAULT_METADATA_MODE));
 					bndInstructions = new BNDInstructions("", null); //$NON-NLS-1$
 					includeSource.setSelection(true);
+					String clipboardError = clipboardParser.getError();
+					if (clipboardError != null) {
+						setMessage(MessageFormat.format(Messages.MavenTargetLocationWizard_11, clipboardError),
```

Comment:
Please just show a MessageDialog.openWarning in that case.

Reply:
I think that's too disturbing to the user. As auto-filling is really optional and we can't assume that users always have valid content a Dialog might really annoy users if triggered by random snippets in the clipboard.

Reply:
The proposed approach is totally alien, it's not used in any other dialog and I'm not certain about how accessible it would be to understand for people using a screenreader. It is probably better to just not show anything then if auto-filling is not applicable, as users are probably not even expecting auto-fill to be happening in that case, a warning is actually more a disturbance than nothing.

Reply:
jgit uses autofilling for the "add repository" dialog, but if you think its better to be silent I'll remove the message, it just showed that if filling is not performed but the user tries to use the feature it is considered a bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=570456 but I think its okay to simply tell people then to check their xml fragment.

Reply:
Auto-filling is nice; however what I don't find good is the temporary warning on startup. As far as I can see, the "Clone Repository" wizard doesn't warn if clipboard content isn't a Git repo URL, it just does nothing, which is good enough.